### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Brian-Doucet**
1. Added new functions to validate parameter input for province/state and country/region
2. Created (2) csv files which contain the list of valid states/provinces and countries
3. Added tests for utils and cases

### CODE REVIEW

File: tests/test_cases.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        "Lat",
+        "Long_",
+        "Confirmed",
+        "Deaths",
+        "Recovered",
+        "Active",
+        "Combined_Key",
+    ]
+    # assertion
+    dt.validate(actual.columns, expected_col_names)
```
Comment (ilyayudkovich) [inline tests/test_cases.py:41]: why not use `actual_col_names_list` here?

File: tests/test_cases.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+# Set of tests for DataFrame filtering on Country/Region and Province/State
+
+# Set up test data
+@pytest.fixture(scope="module")
```
Comment (ilyayudkovich) [inline tests/test_cases.py:47]: love it

File: utils.py [hunk 3]
```diff
@@ -1,8 +1,47 @@
...
+    is_valid_state = state_name in valid_states
+
+    if not is_valid_state:
+        raise Exception(
+            "{} is not valid. Use one of the following values:{}".format(
+                state_name, valid_states
+            )
+        )
+    else:
+        return
```
Comment (ilyayudkovich) [inline utils.py:28]: this doesn't return a boolean, this just returns out of the function without returning anything

**Review by ilyayudkovich — APPROVED** [review]
very good PR! this is using all the concepts we've talked about so far! this is awesome!

File: tests/test_cases.py (hunk 1 above)
Comment (Brian-Doucet) [reply tests/test_cases.py:41]: Makes sense. Validating actual_col_names_list and actual.columns does the same thing.

**Comment by Brian-Doucet** [comment]
All changes reviewed.

**Pull request merged**